### PR TITLE
SEO Boot Implementation Guide

### PR DESCRIPTION
**Pull request by jesuslarareevoo**
SEO Boost Implementation Guide

### CODE REVIEW

File: docs/implementationguides/seo-boost.markdown [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+
+# Overview
+Fresh, crawl-able, and frequently updated reviews can provide great content for search engines to discover and index. The ReevooMark in-line SEO module allows merchants to embed reviews on product pages, or customer experience reviews on any page, easily and efficiently with seamless integration with the standard ReevooMark product.
+
+Our technical integration is:
+
+* Straightforward to setup and implement
+
```
Comment (davidsevcik) [inline docs/implementationguides/seo-boost.markdown]: You don't need these empty lines between points.

File: docs/implementationguides/seo-boost.markdown [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* Supports high volume web sites by automatically handling efficient storage and caching of data
+
+* Automatically handles frequently updating review content
+
+* Integrates seamlessly with the standard ReevooMark product
+
+<br/><br/>
```
Comment (davidsevcik) [inline docs/implementationguides/seo-boost.markdown]: Can we convert all the HTML syntax into markdown?

File: _data/docs.yml [hunk 3]
```diff
@@ -63,3 +63,9 @@
         title: upvote answer
       - id: conversation-downvote-answer
         title: downvote answer
+- id: implementationguides
+  title: Implementation Guides
+  url: implementationguides
+  items:
+  - id: seo-boost
```
Comment (davidsevcik) [inline _data/docs.yml]: I would put SEO Boost page directly under the root. The reevoomark part is also about the implementation.

**Comment by davidsevcik** [comment]
Could you check with John if it's ok to use real customer sites screenshots in the documentation.

File: docs/implementationguides/seo-boost.markdown (hunk 2 above)
Comment (jesuslarareevoo) [inline docs/implementationguides/seo-boost.markdown]: Ok some of it I had to do html on purpose as image scaling and open links in new tab are only supported on html markup.

**Comment by jesuslarareevoo** [comment]
Yeap I sent a link to Zoe and Jon to the documentation running on my local for them to approve as well.

File: _data/docs.yml (hunk 3 above)
Comment (jesuslarareevoo) [inline _data/docs.yml]: Possibly yeah maybe I'll move it to the root

**Pull request merged**